test/kfp-functional-test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# This image has the script to install the base image needed for kfp functional e2e test,
+
+FROM google/cloud-sdk:315.0.0
```

Comment:
Hi @capri-xiyue, sorry I just recalled that the SDK no longer needs `gcloud` as dependency in https://github.com/kubeflow/pipelines/pull/4811.
(so we can also get rid of `gcloud auth activate-service-account` call)

Can you have a try? If that's the case, we can use `python:3.7` image directly.

Reply:
discussed offline